cypress/integration/auth/emailPasswordSignUpForm.js

```diff
@@ -0,0 +1,37 @@
+const FORM_URL = '/iframe.html?id=auth-stories--basic-email-password-sign-up-form';
+const BUTTON = 'button[type="submit"]';
+const SPINNER_BACKDROP = '[data-cy="app-spinner"]';
+const NAME_INPUT = 'input[name="name"]';
+const EMAIL_INPUT = 'input[name="email"]';
+const PASSWORD_INPUT = 'input[name="password"]';
+
+describe('EmailPasswordSignUpForm', () => {
+    beforeEach(() => {
+        cy.visit(FORM_URL);
+
+        cy.server({
+            delay: 1000,
+        });
+        cy.route(
+            'POST',
+            'https://www.googleapis.com/identitytoolkit/v3/relyingparty/signupNewUser?*',
+            [],
+        ).as('firebaseSignUp');
+    });
+
+    it('should request create user endpoint', () => {
+        cy.get(NAME_INPUT).type('Test');
+        cy.get(EMAIL_INPUT).type('test@email.com');
+        cy.get(PASSWORD_INPUT).type('TestPassword');
+        cy.get(BUTTON).click();
+        cy.wait('@firebaseSignUp');
```

Comment:
We should do assertions that check for something that indicates the user is authenticated like the new URL or an element that exists when authenticated.
Ideally, we should start up the emulator and login and then make the assertion but I don't know if right now there is an emulator for auth.

Reply:
As for now, only Database, Firestore, Functions and Hosting are available for emulator. If we are going to test this it will be necesary to hit the real API, and handle the creation and deletion of users each time the tests runs, we could do that but I dont think it's not a good idea, given that we can trust firebase's ```createUserWithEmailAndPassword``` function will correctly authenticate the user when the sign up is correct